--- a/pkg_hash.c	2024-07-03 16:43:56.342845000 +0200
+++ a/pkg_hash.c.patched	2024-11-04 13:44:19.088412000 +0100
@@ -225,10 +225,10 @@
 	  }
      }
 
-     if (matching_pkgs->len > 1)
-	  pkg_vec_sort(matching_pkgs, pkg_name_version_and_architecture_compare);
-     if (matching_apkgs->len > 1)
-	  abstract_pkg_vec_sort(matching_pkgs, abstract_pkg_name_compare);
+//     if (matching_pkgs->len > 1)
+//	  pkg_vec_sort(matching_pkgs, pkg_name_version_and_architecture_compare);
+//     if (matching_apkgs->len > 1)
+//	  abstract_pkg_vec_sort(matching_pkgs, abstract_pkg_name_compare);
 
 /* Here it is usefull, if ( matching_apkgs->len > 1 ), to test if one of this matching packages has the same name of the
    needed package. In this case, I would return it for install, otherwise I will continue with the procedure */
--- a/ipkg_conf.c	2024-07-03 16:43:56.294844000 +0200
+++ a/ipkg_conf.c.patched	2024-11-04 13:36:57.981942000 +0100
@@ -548,19 +548,19 @@
 	  if (strcmp(type, "option") == 0) {
 	       ipkg_conf_set_option(options, name, value);
 	  } else if (strcmp(type, "src") == 0) {
-	       if (!nv_pair_list_find(pkg_src_list, name)) {
+//	       if (!nv_pair_list_find(pkg_src_list, name)) {
 		    pkg_src_list_append (pkg_src_list, name, value, extra, 0);
-	       } else {
-		    ipkg_message(conf, IPKG_ERROR, "ERROR: duplicate src declaration.  Skipping:\n\t src %s %s\n",
-				 name, value);
-	       }
+//	       } else {
+//		    ipkg_message(conf, IPKG_ERROR, "ERROR: duplicate src declaration.  Skipping:\n\t src %s %s\n",
+//				 name, value);
+//	       }
 	  } else if (strcmp(type, "src/gz") == 0) {
-	       if (!nv_pair_list_find(pkg_src_list, name)) {
+//	       if (!nv_pair_list_find(pkg_src_list, name)) {
 		    pkg_src_list_append (pkg_src_list, name, value, extra, 1);
-	       } else {
-		    ipkg_message(conf, IPKG_ERROR, "ERROR: duplicate src declaration.  Skipping:\n\t src %s %s\n",
-				 name, value);
-	       }
+//	       } else {
+//		    ipkg_message(conf, IPKG_ERROR, "ERROR: duplicate src declaration.  Skipping:\n\t src %s %s\n",
+//				 name, value);
+//	       }
 	  } else if (strcmp(type, "dest") == 0) {
 	       nv_pair_list_append(tmp_dest_nv_pair_list, name, value);
 	  } else if (strcmp(type, "lists_dir") == 0) {
--- a/ipkg_download.c	2024-07-03 16:43:56.238843000 +0200
+++ a/ipkg_download.c.patched	2024-11-04 13:27:40.360668000 +0100
@@ -35,7 +35,7 @@
 		value2 = malloc(1);
 		if(value2 == NULL)
 		{
-			err("no memory");
+//			err("no memory");
 			return NULL;
 		}
 		free2 = 1;
@@ -47,7 +47,7 @@
 		value1 = malloc(1);
 		if(value1 == NULL)
 		{
-			err("no memory");
+//			err("no memory");
 			return NULL;
 		}
 		free1 = 1;
@@ -57,7 +57,7 @@
 	buf = malloc(strlen(value1) + strlen(value2) + 1);
 	if(buf == NULL)
 	{
-		err("no memory");
+//		err("no memory");
 		return NULL;
 	}
