--- edid-decode/Makefile	2022-10-21 11:28:19.374625586 +0200
+++ edid-decode/Makefile.patched	2022-10-21 11:35:57.939701315 +0200
@@ -18,7 +18,7 @@
 OBJECTS := $(patsubst %.cpp, %.o, $(SOURCES))
 EMOBJECTS := $(patsubst %.cpp, emscripten/%.o, $(SOURCES))
 
-WARN_FLAGS = -Wall -Wextra -Wno-missing-field-initializers -Wno-unused-parameter -Wimplicit-fallthrough
+WARN_FLAGS = -Wall -Wextra -Wno-missing-field-initializers -Wno-unused-parameter
 
 all: edid-decode
